src/components/forms/PhoneInput.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from "react"
+
+import {
+  Input,
+  InputGroup,
+  InputLeftAddon,
+  InputRightAddon,
+  InputProps as ChakraInputProps,
+  Select,
+} from "@chakra-ui/react"
+
+export type E164Number = string
+
+export enum CountryCode {
+  AU = "AU",
+  US = "US",
+}
+
+export interface CountryData {
+  name: string
+  emoji: string
+  countryCode: string
+  // TODO use template literal type once we've migrated away from tsdx
```

Comment:
we can action this todo now! calling code's type can move back to a string literal template

e.g.

```typescript
callingCode: `+${string}`
```